.scripts/verify-pull-request.yaml

```diff
@@ -0,0 +1,89 @@
+# Node.js
+# Build a general Node.js project with npm.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+trigger: none
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: '10.x'
+  displayName: 'Install Node.js'
+
+- script: |
+    npm install -g npm
+    npx @microsoft/rush update
+    npx @microsoft/rush rebuild
+
+    # Tests are disabled for now, failures to be investigated (Azure/autorest#3400)
+    #npx @microsoft/rush test
+
+  displayName: 'Rush install, build and test'
+
+- script : |
+    npm install @microsoft.azure/autorest.testserver --no-save
+    npm install -g @autorest/compare
+  displayName: Install autorest-compare
+
+- script : |
+    export AUTOREST_TYPESCRIPT=@autorest/typescript@0.1.0-dev.20200203.1
+    autorest-compare --typescript \
+                     --spec-path:azure-report.json \
```

Comment:
I'm designing a JSON config format which will eliminate the need for such a long command line invocation.

Reply:
Maybe use YAML; all valid JSON is valid YAML, and sometimes it's nice to have comments.  yaml-js works nice for parsing.

Reply:
Thanks, will do!

Reply:
Yeah, a config file would be really nice -- with this wrapped up in a pipeline, we can't really leverage this on the dev machine without cut/paste.

Reply:
Yep.  It would also be super difficult to regenerate baseline output consistently without a config file.  I'm going to implement my plan this morning and send out a PR before lunch.